litmus/VULKAN/DR/privpo-filter.litmus

```diff
@@ -0,0 +1,13 @@
+Vulkan privpo
```

Comment:
This test is wrong. [The original one](https://github.com/KhronosGroup/Vulkan-MemoryModel/blob/66c31d9cfcc03b1f5231779fbf3c42dc983b62e2/alloy/tests/privpo.test) has only one thread. Please update both instances of the test, i.e. `/litmus/VULKAN` and `/litmus/VULKAN/DR`